openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -60,6 +60,7 @@ class LearningContext(TimeStampedModel):
     title = models.CharField(max_length=255)
     published_at = models.DateTimeField(null=False)
     published_version = models.CharField(max_length=255)
+    course_visibility = models.CharField(max_length=255)
```

Comment:
I'm of two minds on this. On the one hand, `course_visibility` is a course-centric concept, and may not belong in something that's meant to be generic across all learning contexts (like learning pathways and content libraries). That being said, we could introduce a more generic concept of `content_visibility`, which would still be public/public_outline/private and leave it in this model–with the understanding being that content libraries and learning pathways might have equivalent requirements.

What do you think?

Reply:
Or maybe some other name... (sigh)... names are hard...

Reply:
Actually, given that beta user support is also going to need a course-specific attribute, I'm leaning more towards new model and not trying to generalize the concept yet.